### PR TITLE
Solve problem with php version #427

### PR DESCRIPTION
**Pull request by aicelm**
I added this condition to solve the problem with php version 7.1

### CODE REVIEW

**Comment by aicelm** [comment]
Hi Bjorn, 

I don't know if I understood very well.
Is it better that I correct the code so that it doesn't error in the version that fail or I should add a version to send a error message, as in the example you sent me? 

> For an example, see https://github.com/Esri/resource-proxy/blob/master/PHP/proxy-verification.php#L145

The version that is having problems is the 7.1 or some more?

Thank you,
Regards!

**Comment by bsvensson** [comment]
Hi Leticia,
The version that the proxy fails with isn't just 7.1, it's 7.1 plus any [future 7.2+](https://wiki.php.net/todo/php72) version as well.

So I suggest that instead of checking specifically for "7.1" in the string, it would be better to check if it's version "7.1" or later.  This can be done using http://php.net/version_compare, maybe:
`if (version_compare(PHP_VERSION, '7.1.0') >= 0) {`  // (untested)

Does that make more sense?

File: PHP/proxy.php [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (dasa) [inline PHP/proxy.php]: This new line should probably be removed.

**Comment by bsvensson** [comment]
https://github.com/Esri/resource-proxy/pull/443 has been merged which adds initial GET support for PHP 7.1+.  I don't think this specific PR is needed.

**Pull request closed without merging**